<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>581</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>摄像头参数</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>561</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>摄像头设备</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboCamera"/>
      </item>
      <item>
       <widget class="QCheckBox" name="checkStillImage">
        <property name="text">
         <string>支持抓图</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkVideo">
        <property name="text">
         <string>支持录像</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>561</width>
       <height>33</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="chkMute">
        <property name="text">
         <string>录像时静音</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnVideoSaveFile">
        <property name="text">
         <string>视频保存文件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editOutputFile">
        <property name="text">
         <string>/Users/wangjiankang/Pictures/video.mp4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LabDuration">
        <property name="text">
         <string>录制时间</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>301</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>摄像头预览</string>
    </property>
    <widget class="QCameraViewfinder" name="viewFinder" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>301</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>130</y>
      <width>271</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>抓取图片</string>
    </property>
    <widget class="QLabel" name="LabCapturedImage">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>271</width>
       <height>201</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actStartCamera"/>
   <addaction name="actStopCamera"/>
   <addaction name="actCapture"/>
   <addaction name="separator"/>
   <addaction name="actVideoRecord"/>
   <addaction name="actVideoStop"/>
   <addaction name="separator"/>
   <addaction name="actQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actStartCamera">
   <property name="text">
    <string>开启摄像头</string>
   </property>
   <property name="toolTip">
    <string>开启摄像头</string>
   </property>
  </action>
  <action name="actStopCamera">
   <property name="text">
    <string>关闭摄像头</string>
   </property>
   <property name="toolTip">
    <string>关闭摄像头</string>
   </property>
  </action>
  <action name="actVideoRecord">
   <property name="text">
    <string>开始录像</string>
   </property>
   <property name="toolTip">
    <string>开始录像</string>
   </property>
  </action>
  <action name="actCapture">
   <property name="text">
    <string>抓图</string>
   </property>
   <property name="toolTip">
    <string>抓图</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actVideoStop">
   <property name="text">
    <string>停止录像</string>
   </property>
   <property name="toolTip">
    <string>停止录像</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCameraViewfinder</class>
   <extends>QWidget</extends>
   <header>qcameraviewfinder.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
